File: 코딩 기초 트레이닝/수열과 구간 쿼리 1-p181883/수열과 구간 쿼리 1.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181883

function solution(arr, queries) {
    // s <= i <- e 라면 arr[i] + 1
    const answer = [...arr];

    queries.forEach((q, index) => {
        arr.forEach((a, i) => {
            if(i >= q[0] && i <= q[1]) {
                answer[i] += 1;
            }
        });
    });


    return answer;
}

File: 코딩 기초 트레이닝/수열과 구간 쿼리 1-p181883/수열과 구간 쿼리 1.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181883

def solution(arr, queries):
    answer = arr

    for index, query in enumerate(queries):
        for i, a in enumerate(arr):
            if i >= query[0] and i <= query[1]:
                answer[i] += 1

    return answer
